tests/tests_create_raid_volume_then_remove.yml

```diff
@@ -56,3 +57,195 @@
             state: absent
 
     - include_tasks: verify-role-results.yml
+
+    # For testing raid 1
+    - include_tasks: get_unused_disk.yml
+      vars:
+        max_return: 2
+
+    - name: Create a RAID1 device mounted on "{{ mount_location }}"
+      include_role:
+        name: storage
+      vars:
+        safe_mode: false
```

Comment:
This should not do anything, because the variable is actually called `storage_safe_mode`, `safe_mode` should not be recognized. It should be also unneeded, since `storage_safe_mode: false` is set at the beginning of the playbook.